Index: Makefile.PL
===================================================================
--- a/Makefile.PL	(revision 390)
+++ b/Makefile.PL	(revision 391)
@@ -33,7 +33,7 @@
     ],
     ["Client HTTPS support","SOAP::Transport::HTTPS::Client",{"Crypt::SSLeay" => 0},0],
     ["Client SMTP/sendmail support","SOAP::Transport::MAILTO::Client",{"MIME::Lite" => 0},1],
-    ["Client FTP support","SOAP::Transport::FTP::Client",{"SOAP::Transport::FTP" => 0.712},0],
+    ["Client FTP support","SOAP::Transport::FTP::Client",{"SOAP::Transport::FTP" => 0.711},0],
     ["Client TCP support","SOAP::Transport::TCP::Client",{"SOAP::Transport::TCP" => 0.714},0],    
     ["Standalone HTTP server","SOAP::Transport::HTTP::Daemon",{"HTTP::Daemon" => 0},1],
     ["Apache/mod_perl server","SOAP::Transport::HTTP::Apache",{"Apache" => 0},0],
--- a/META.yml	2011-11-07 09:50:19.224142630 +0000
+++ b/META.yml	2011-11-07 09:50:27.629037553 +0000
@@ -24,7 +24,7 @@
     Net::POP3:            0
     perl:                 5.006
     Scalar::Util:         0
-    SOAP::Transport::FTP:  0.712
+    SOAP::Transport::FTP:  0.711
     SOAP::Transport::JABBER:  0.712
     Task::Weaken:         0
     Test::More:           0
